systers_portal/templates/contact.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
Hey @Isabel-S! Thank you for so much work! If you plan to incorporate the page as an Django template please update this pull request and reopen, closing this for now, just to keep things a bit clean.